src/Features/Core/Portable/FeaturesResources.resx

```diff
@@ -234,11 +234,11 @@
   <data name="GenerateLocal" xml:space="preserve">
     <value>Generate local '{0}'</value>
   </data>
-  <data name="GenerateForInNewFile" xml:space="preserve">
-    <value>Generate {0} for '{1}' in '{2}' (in new file)</value>
+  <data name="Generate_0_1_in_new_file" xml:space="preserve">
```

Comment:
I kind of like them. Given that they'll only ever be prefixes with a ```*Resources``` type, I think it'll look fine.